bin/up

```diff
@@ -9,6 +9,12 @@ if type -a minikube >/dev/null 2>/dev/null; then
     eval ${dockerenv}
 fi
 
+GIT_ROOT="${GIT_ROOT:-$(git rev-parse --show-toplevel)}"
```

Comment:
I just noticed this introduces `GIT_ROOT` to have the absolute path available, but most of the script still uses relative paths (line 20, ...).
We could remove `GIT_ROOT` here and in `bin/test-integration`, too.